scripts/component_generation/main.go

```diff
@@ -17,15 +15,28 @@ import (
 
 	"github.com/layer5io/meshkit/models/meshmodel/core/v1alpha1"
 	"github.com/layer5io/meshkit/utils/artifacthub"
-	"golang.org/x/oauth2/google"
-	"google.golang.org/api/option"
-	"google.golang.org/api/sheets/v4"
 	"gopkg.in/yaml.v3"
 )
 
 const dumpFile = "./dump.csv"
 const COLUMNRANGE = "!A:AH3" //Update this on addition of new columns
 const APPENDRANGE = "!A4:V4"
+const sheetID = 0
+
+var spreadsheetID = "1DZHnzxYWOlJ69Oguz4LkRVTFM79kC2tuvdwizOJmeMw"
+
+type componentWrapper struct {
```

Comment:
@MUzairS15 @Yashs911 am I make sense that this point to Meshery’s preferred Helmrepo connection provider: the model for Flux.